http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppControlledResourceComponent.scala

```diff
@@ -3,50 +3,64 @@ package org.broadinstitute.dsde.workbench.leonardo.db
 import LeoProfile.api._
 import LeoProfile.mappedColumnImplicits._
 import org.broadinstitute.dsde.workbench.leonardo.WsmControlledResourceId
+import ca.mrvisser.sealerate
 
 import scala.concurrent.ExecutionContext
 
-case class AppControlledResourceRecord(appId: Long, resourceId: WsmControlledResourceId, resourceType: WsmResourceType)
+sealed abstract class AppControlledResourceStatus
+object AppControlledResourceStatus {
+  case object Created extends AppControlledResourceStatus {
+    override def toString: String = "CREATED"
+  }
+  case object Deleted extends AppControlledResourceStatus {
+    override def toString: String = "DELETED"
+  }
+
+  def values: Set[AppControlledResourceStatus] = sealerate.values[AppControlledResourceStatus]
+
+  def stringToObject: Map[String, AppControlledResourceStatus] = values.map(v => v.toString -> v).toMap
+}
+case class AppControlledResourceRecord(appId: Long,
+                                       resourceId: WsmControlledResourceId,
+                                       resourceType: WsmResourceType,
+                                       status: AppControlledResourceStatus
+)
 
 class AppControlledResourceTable(tag: Tag) extends Table[AppControlledResourceRecord](tag, "APP_CONTROLLED_RESOURCE") {
   def appId = column[Long]("appId")
   def resourceId = column[WsmControlledResourceId]("resourceId")
   def resourceType = column[WsmResourceType]("resourceType")
+  def status = column[AppControlledResourceStatus]("status", O.Length(254))
 
   def * =
-    (appId, resourceId, resourceType) <> (AppControlledResourceRecord.tupled, AppControlledResourceRecord.unapply)
+    (appId,
+     resourceId,
+     resourceType,
+     status
+    ) <> (AppControlledResourceRecord.tupled, AppControlledResourceRecord.unapply)
 }
 
 object appControlledResourceQuery extends TableQuery(new AppControlledResourceTable(_)) {
-  def save(appId: Long, resourceId: WsmControlledResourceId, resourceType: WsmResourceType): DBIO[Int] =
-    appControlledResourceQuery += AppControlledResourceRecord(appId, resourceId, resourceType)
-
-  def getWsmRecordForApp(appId: Long, resourceType: WsmResourceType): DBIO[Option[AppControlledResourceRecord]] =
```

Comment:
@dvoet is there a reason why this function is being removed? I was making use of it and I am now wondering what i should replace it with :)